tests/unit/components/cylc/tree/tree.spec.js

```diff
@@ -289,8 +289,6 @@ describe('CylcTree', () => {
       })
       cylcTree.addFamilyProxy(familyProxy1)
       cylcTree.addFamilyProxy(familyProxy1)
-      console.log(cyclePoint2.id)
-      console.log(cylcTree.root.children[0])
```

Comment:
My bad, not necessary for this test :+1:  (we do have some `console.log` or `console.error` being tested, to confirm warnings are raised, etc… but here was just debugging I forgot to remove).